coc.md

```diff
@@ -6,7 +6,7 @@ Be kind and courteous. Being mean or rude scares people off and makes people not
 
 Low-effort criticism is for the benefit of the person delivering it, not the person receiving it.
 
-Don't describe things as "easy" or "trivial". You'll making people feel terrible for having to work hard for their progress. Slow learners are often the most thorough learners, this is something to celebrate!
```

Comment:
-Don't describe things as "easy" or "trivial". You'll make people feel terrible for having to work hard for their progress. Slow learners are often the most thorough learners, this is something to celebrate!